packages/calcite-components/src/assets/styles/includes.scss

```diff
@@ -152,3 +152,40 @@
     }
   }
 }
+
+@mixin close-button($size: "var(--calcite-internal-close-size, 1.5rem /* 24px */)", $padding: "0") {
```

Comment:
Sidebar: we'll need to consolidate this with the `x-button` mixin above. Related https://github.com/Esri/calcite-design-system/issues/6640.